.github/workflows/ci.yml

```diff
@@ -61,20 +55,17 @@ jobs:
       AWS_ACCESS_KEY_ID: test
       AWS_SECRET_ACCESS_KEY: test
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions-rs/toolchain@v1
-      with:
-        components: rustfmt, clippy
-        # Pinned to the commit hash of v2.2.1
-    - uses: Swatinem/rust-cache@6fd3edff6979b79f87531400ad694fb7f2c84b1f
+    - uses: actions/checkout@v4
+    - name: Install rust toolchain
+      run: |
+        rustup set profile minimal
+        rustup component add rustfmt clippy
+    - uses: Swatinem/rust-cache@640a22190e7a783d4c409684cea558f081f92012
+        # This is the most recent commit as of 2024-03-22.
+        # Also, it uses Node.js 20.
       with:
         shared-key: build-and-test-on-windows
-    - uses: actions-rs/cargo@v1
-      with:
-          # Currently, we only conduct snapshot tests because GitHub Actions does not support containers in Windows.
```

Comment:
We need to remain this comment.

Reply:
Thank you for finding this. I have fixed it.